src/modules/app/route-configs/CategoriesRouteConfig.js

```diff
@@ -1,7 +1,7 @@
 // @flow
 
 import { RouteConfig } from './RouteConfig'
-import type { Dispatch, GetState, Route } from 'redux-first-router'
+import type { Dispatch, GetState, Route } from 'redux-first-router/dist/flow-types'
```

Comment:
```suggestion
import type { Route } from 'redux-first-router'
```
We don't need to import Dispatch and GetState, since flow know their type by the Route type

Reply:
Yeah ok, but one comment for this would have been enough. :D